WooCommerce/WooCommerceTests/ViewRelated/CardPresentPayments/CardPresentPaymentsOnboardingUseCaseTests.swift

```diff
@@ -179,6 +179,96 @@ class CardPresentPaymentsOnboardingUseCaseTests: XCTestCase {
         XCTAssertNotEqual(state, .countryNotSupported(countryCode: "CA"))
     }
 
+    func test_onboarding_does_not_return_country_unsupported_with_uk_when_neither_wcpay_nor_stripe_plugin_installed() {
```

Comment:
Should we change the test naming from `_plugin_installed` to `_plugin_active`? As we're checking for `SitePluginStatusEnum.active` case, not its installation. Same for the other tests